src/main.rs

```diff
@@ -34,12 +36,16 @@ enum Commands {
 fn main() -> Result<()> {
     let cli = Cli::parse();
 
-    eprintln!("Unpacking relations from {:?}", cli.in_file);
+    SimpleLogger::new()
```

Comment:
Können wir die logs aber an sterr umleiten, damit die stdout nur die Daten enthält?

Reply:
Logs gehen nach `stderr` mit dem `stderr` feature von `simple_logger` in `Cargo.toml`,
```
simple_logger = { version = "4.2.0", features = ["stderr"] }
```